packages/fakta-beregning/src/utils/aksjonspunktUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { isAksjonspunktOpen } from '@navikt/ft-kodeverk';
+import { BeregningAvklaringsbehov } from '@navikt/ft-types';
+
+import { FaktaBeregningAvklaringsbehovCode } from '../typer/interface/FaktaBeregningAvklaringsbehovCode';
+
+export const hasAksjonspunkt = (aksjonspunktKode: string, avklaringsbehov: BeregningAvklaringsbehov[]): boolean =>
+  avklaringsbehov.some(ap => ap.definisjon === aksjonspunktKode);
+
+export const isAksjonspunktClosed = (avklaringsbehov: BeregningAvklaringsbehov[]): boolean => {
+  const relevantAp = avklaringsbehov.filter(
```

Comment:
Litt forvirrande begrepsbruk her. Eit avklaringsbehov blir til eit aksjonspunkt. Burde vel vore aksjonspunkt gjennom heile, heilt frå backend? (Mogleg Petter kan svara på kvifor det ikkje er sånn)

Reply:
Det ble i sin tid kalt avklaringsbehov i backend for å ha et klarere skille mellom k9sak og ft-kalkulus, og gjøre klar at de er forskjellige ting (et aksjonspunkt er knyttet til en behandling, et avklaringspunkt er knyttet til et beregningsgrunnlag). Kunne med fordel gjennomgått beregningskoden frontend og fått det konsist.